ethcore/evm/src/instructions.rs

```diff
@@ -165,436 +453,189 @@ impl InstructionInfo {
-		arr[DUP16 as usize] =			InstructionInfo::new("DUP16",			16, 17, GasPriceTier::VeryLow);
-		arr[SWAP1 as usize] =			InstructionInfo::new("SWAP1",			2, 2, GasPriceTier::VeryLow);
-		arr[SWAP2 as usize] =			InstructionInfo::new("SWAP2",			3, 3, GasPriceTier::VeryLow);
-		arr[SWAP3 as usize] =			InstructionInfo::new("SWAP3",			4, 4, GasPriceTier::VeryLow);
-		arr[SWAP4 as usize] =			InstructionInfo::new("SWAP4",			5, 5, GasPriceTier::VeryLow);
-		arr[SWAP5 as usize] =			InstructionInfo::new("SWAP5",			6, 6, GasPriceTier::VeryLow);
-		arr[SWAP6 as usize] =			InstructionInfo::new("SWAP6",			7, 7, GasPriceTier::VeryLow);
-		arr[SWAP7 as usize] =			InstructionInfo::new("SWAP7",			8, 8, GasPriceTier::VeryLow);
-		arr[SWAP8 as usize] =			InstructionInfo::new("SWAP8",			9, 9, GasPriceTier::VeryLow);
-		arr[SWAP9 as usize] =			InstructionInfo::new("SWAP9",			10, 10, GasPriceTier::VeryLow);
-		arr[SWAP10 as usize] =			InstructionInfo::new("SWAP10",			11, 11, GasPriceTier::VeryLow);
-		arr[SWAP11 as usize] =			InstructionInfo::new("SWAP11",			12, 12, GasPriceTier::VeryLow);
-		arr[SWAP12 as usize] =			InstructionInfo::new("SWAP12",			13, 13, GasPriceTier::VeryLow);
-		arr[SWAP13 as usize] =			InstructionInfo::new("SWAP13",			14, 14, GasPriceTier::VeryLow);
-		arr[SWAP14 as usize] =			InstructionInfo::new("SWAP14",			15, 15, GasPriceTier::VeryLow);
-		arr[SWAP15 as usize] =			InstructionInfo::new("SWAP15",			16, 16, GasPriceTier::VeryLow);
-		arr[SWAP16 as usize] =			InstructionInfo::new("SWAP16",			17, 17, GasPriceTier::VeryLow);
-		arr[LOG0 as usize] =			InstructionInfo::new("LOG0",			2, 0, GasPriceTier::Special);
-		arr[LOG1 as usize] =			InstructionInfo::new("LOG1",			3, 0, GasPriceTier::Special);
-		arr[LOG2 as usize] =			InstructionInfo::new("LOG2",			4, 0, GasPriceTier::Special);
-		arr[LOG3 as usize] =			InstructionInfo::new("LOG3",			5, 0, GasPriceTier::Special);
-		arr[LOG4 as usize] =			InstructionInfo::new("LOG4",			6, 0, GasPriceTier::Special);
-		arr[CREATE as usize] =			InstructionInfo::new("CREATE",			3, 1, GasPriceTier::Special);
-		arr[CALL as usize] =			InstructionInfo::new("CALL",			7, 1, GasPriceTier::Special);
-		arr[CALLCODE as usize] =		InstructionInfo::new("CALLCODE",		7, 1, GasPriceTier::Special);
-		arr[RETURN as usize] =			InstructionInfo::new("RETURN",			2, 0, GasPriceTier::Zero);
-		arr[DELEGATECALL as usize] =	InstructionInfo::new("DELEGATECALL",	6, 1, GasPriceTier::Special);
-		arr[STATICCALL as usize] =		InstructionInfo::new("STATICCALL",		6, 1, GasPriceTier::Special);
-		arr[SUICIDE as usize] = 		InstructionInfo::new("SUICIDE",			1, 0, GasPriceTier::Special);
-		arr[CREATE2 as usize] = 		InstructionInfo::new("CREATE2",			3, 1, GasPriceTier::Special);
-		arr[REVERT as usize] =			InstructionInfo::new("REVERT",			2, 0, GasPriceTier::Zero);
+	static ref INSTRUCTIONS: [Option<InstructionInfo>; 0x100] = {
+		let mut arr = [None; 0x100];
+		arr[STOP as usize] =			Some(InstructionInfo::new("STOP",			0, 0, GasPriceTier::Zero));
```

Comment:
This tab formatting displays correctly on my Emacs, but looks like it doesn't work on Github interface.
I'm thinking about maybe it's better to just use a single space as separator -- we don't gain much readability by using multiple tabs.